pkgs/servers/mail/listmonk/default.nix

```
@@ -0,0 +1,54 @@
+{ lib, buildGoModule, fetchFromGitHub, callPackage, stuffbin }:
+let
+  frontend = callPackage ./frontend.nix { };
+  vfsMappings = [
+    "config.toml.sample"
+    "schema.sql"
+    "queries.sql"
+    "static/public:/public"
+    "static/email-templates"
+    "${frontend}:/admin"
```

if we move this into the let in then we can use the following:

```suggestion
    "${passthru.frontend}:/admin"
```

---

Done.